### ISSUE TITLE
FutureWarning: Objects based on the `Geometry` class will deprecated and removed in a future version of libpysal

### ISSUE THREAD

**Issue by catamech-aubay** [pysal/libpysal#769]
I was searching for an alternative for the 'Geometry' class that will be deprecated and removed in a future version (undefined), I need to find a solution to be able to use 'Point' and 'Polygon' shapes precisely, but I couldn't find any documentation or previous issue concerning this subject, any idea when will it be deprecated and if there is any suggested solution ?

**Comment by ljwolf** [comment]
I would recommend considering [pygeoif](https://github.com/cleder/pygeoif/) for this functionality. PySAL's geometry types are not OGC-compliant, and were designed to match now outdated reference implementations of ESRI geometries.

**Comment by martinfleis** [comment]
I tend to disagree a bit in here. Installing shapely has not been an issue lately and while it is not pure python it is way more feature complete. And importantly, it is what most of the ecosystem uses. Let's pick it up on Thursday during the dev call to formulate some recommendations on where to point people. I have always assumed we do deprecate and suggest replacement by shapely as that is what we do internally but that might have been a wrong assumption.

**Comment by ljwolf** [comment]
All of that is true. But, pysal geometry classes were written to avoid depending on geos. Hence I'd think a successor recommendation should include a non-geos option.I think if people can get it, shapely is _the_ thing to use!
